lib/versioning/node/index.ts

```diff
@@ -27,8 +30,26 @@ function getNewValue({
 
 export { isValid };
 
+const currentTime = Date.now();
```

Comment:
Won't this be fixed once the app loads? e.g. if it starts running on the day before LTS and is running a day after, then it won't detect that LTS has been reached? I think we need to use Date.now() for every call.

Reply:
yes, we need to always check now, because the app is long running

Reply:
Right, just wanted to "debounce" it a little bit, didn't think about long-run :)